css/settings--jekyll.scss

```diff
@@ -0,0 +1,35 @@
+---
+---
+
+/*
+  As jekyll doesn't provide a way to include liquid templating constructs
+  in scss partials in the `_sass` folder, this file will serve for that
+  purpose. If there's ever the need to include a jekyll variable on a
+  stylesheet, it'll be done on this file. It should be kept organized
+  with the same sections/components that are on the `_sass` folder.
+*/
+
+@import "../node_modules/bootstrap/scss/variables";
+@import "../node_modules/bootstrap/scss/mixins";
+
+/**
+ * Services Section
+ * _sass/_components--service.scss
+ *
+ * Bullet images for the lists on each service page
+ */
+
+.service--rails .service-how-list-item::before {
```

Comment:
Oh got it, so what is the idea of running the jekyll code here? why not target the image with the class?